tests/sles4sap/sapconf.pm

```diff
@@ -35,6 +35,52 @@ sub check_profile {
       unless ($output =~ /Current active profile: $profile/);
 }
 
+sub run_developers_tests {
+    my $devel_repo = 'https://gitlab.suse.de/AngelaBriel/sapconf-test/repository/master/archive.tar.gz';
+    my $log        = '/tmp/sapconf_test.log';
+
+    # Download and unpack the test scripts supplied by the developers
+    # Record soft failure and continue if it can not be downloaded
+    my $ret    = script_run "curl -k $devel_repo > /tmp/sapconf-test.tar.gz";
+    my $output = script_output "file /tmp/sapconf-test.tar.gz";
+    unless (defined $ret and $ret == 0 and $output =~ /gzip compressed data/) {
```

Comment:
again it is too much , if ($ret != 0) will give you same result but save time for person who will try to read and understand what is actually happening here

Reply:
correction - @coolo convince me that we need to check file content because curl can fool us. But I still think that 3 checks is way too much here and you can achieve same result with one check :

```
script_run "curl -k $devel_repo > /tmp/sapconf-test.tar.gz";
$output = script_output "file /tmp/sapconf-test.tar.gz";
if( $output !~ /gzip compressed data/) {
......
```

Reply:
you can also assert_script_run("curl $devel_repo | tar xvf -"); this will always fail if it fails :)

Reply:
I like the `curl $devel_repo | tar -xvf -` option, but will not implement it with assert because I'd rather soft fail the test if I can not download the developer's package. Thanks.